### PR TITLE
Feature fullWeb

### PR DESCRIPTION
**Pull request by tufoinnkuo10**
In this project; I
- created Two new pages (the Home and the Quote page)
- All pages are linked together in the header
- Installed and use the React Router
- Used React basic styling (imported the CSS)
- Updated the Readme page

### CODE REVIEW

**Comment by tufoinnkuo10** [comment]
My eslint ran on vscode with no errors but github was bringing up one error.I had to disable eslint on the app.js home import  line(line5). you can see from the number of times I tried to fix the error in my commits. I am open to any solution though

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @tufoinnkuo10 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
-import react from 'react';
+import { Switch, Route } from 'react-router-dom';
 import Calculator from './components/calculator';
+import Header from './components/Header';
+import Quote from './components/QuotePage';
+/* eslint-disable-next-line */
```
Comment (mikenath223) [inline src/App.js]: - [x] Please remove this eslint disable rule. You can refer to [this link](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md) for an idea on how to fix it.

File: README.md [hunk 2]
```diff
@@ -4,12 +4,13 @@
...
-> This is a Calculator app that solves basic math problems. Built with React
-> ![demo-gif](./images/calcdemo.gif)
+> This is a SPA Calculator app that solves basic math problems. Built with React
+It consist of a Home/Calculator and Quote page
+> ![demo-gif](./images/calc.gif)
 
 
 Additional description about the project and its features.
-
+- it has three pages linked together
```
Comment (mikenath223) [inline README.md]: - [x] Please update the readme following [this template](https://github.com/microverseinc/readme-template). Update the project name and add other relevant sections.

**Review by rukundoeric — APPROVED** [review]
Hi @tufoinnkuo10,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**